app/assets/javascripts/table-select-all.es6

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function() {
+  if ($('.select-clear-all').length) {
+    function resetSelectClear(table) {
+      const link = table.find('.select-clear-all');
+      link.removeClass('select-all clear-all');
+
+      if (table.find('input:checked').length) {
+        link.addClass('clear-all').text(link.data('copy-clear'));
+      }
+      else {
+        link.addClass('select-all').text(link.data('copy-select'));
+      }
```

Comment:
I think this means that the select-all option is only available if no check boxes are checked. That is, if you check one check box, you can only clear all. Is that correct?

Reply:
Yes, that's correct